### ISSUE TITLE
Import Error when trying to run run_train.py

### ISSUE THREAD

**Issue by omerbakler** [ziegler-ingo/cleavage_benchmark#1]
When I try to run this file I get the following message:

ImportError: cannot import name 'FileLoader' from 'torch.utils.data.datapipes.iter' (../envs/cleavage_benchmark/lib/python3.10/site-packages/torch/utils/data/datapipes/iter/__init__.py)

When I go to the specific repository and look at the __init__ file, there is indeed no FileLoader module. I think it is related to the fact I changed the torchdata version to 0.3.0a0, without +1546ecf, because it is not found. Furthermore, when I try to look for this very specific version on google, the only page it is mentioned only is this github.

Additionally, It would be nice to get some instructions on the proper order to run the code files. 

Thank you.

**Comment by ziegler-ingo** [comment]
It can very well be that `init.py` files are empty, but the underlying repository structure should still take care of proper working imports. 
I am not 100% sure why you are running into this problem, but you should be able to install a current version of torch, which should install the correct version of torchdata as a dependency. In my code, I am just using the default torch dataloaders, so no special imports are required anywhere (see [here](https://github.com/ziegler-ingo/cleavage_benchmark/blob/master/code/loaders.py#L6)). I just exported my environment from the moment I installed everything, so I am also not sure how I got to the `torchdata==0.3.0a0+1546ecf` version.

As for running the files: You can start a training run using the provided configs by running on the command line from the source directory e.g. `python3 code/run_train.py $(cat code/run_configs/c_bilstm.cfg)`. All model options we tested have the config files structured [according to this naming scheme](https://github.com/ziegler-ingo/cleavage_benchmark/tree/master?tab=readme-ov-file#naming-structure-of-runtime-configs).

Let me know if you have any other questions.

**Comment by omerbakler** [comment]
Still. I don't understand why the problem continue to occur. Is this code runs smoothly for you now? I just to make sure it is not a general problem with the dependencies.

**Comment by ziegler-ingo** [comment]
Yes, I just tested the code snippet `python3 code/run_train.py $(cat code/run_configs/c_bilstm.cfg`  with pytorch 2.2.1 and python 3.11.0, it works fine. Maybe you can just create a completely new environment and try again (without installing my mentioned dependencies from the `environment.yml` file)?
